src/ui/components/forms/VPNTextArea.qml

```diff
@@ -34,7 +34,8 @@ Item {
         }
 
         TextArea.flickable: TextArea {
-            property var maxCharacterCount: 1000
+            property int maxCharacterCount: 1000
+            property int textLength: textArea.text.length
```

Comment:
why a property here for this? You can have this as a variable in the function.

Reply:
Storing the text length in a property here lets us compare the previous text with the updated text in `handleOnTextChanged`. Doing this in the function would mean that `textLength` and `updatedTextLength` always have the same value.